src/models/relationshipType.js

```diff
@@ -21,11 +21,14 @@ import {camelToSnake, snakeToCamel} from '../util';
 
 export default function relationshipType(bookshelf) {
 	const RelationshipType = bookshelf.Model.extend({
+		attribute() {
```

Comment:
This would be more descriptive I think:
```suggestion
		attributeTypes() {
```